docs/pages/access-controls/idps/saml.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Using Teleport as a SAML identity provider
```

Comment:
This page kind of seems stuck halfway between being a guide and being a reference page.

In places, it reads like a guide, because it mentions steps in a specific sequence. It's missing some of the guidelines we use for guides though (namely the 'Step 1/N' section titles).

I would consider breaking this up into two separate pages:

- A reference page with an overview of the feature, why you might want to use it, and how it works.
- A step-by-step guide for setting it up with a particular sample app.

This would allow you to be specific in the guide and not say things like "the process varies greatly, so consult the documentation of your app"

Reply:
I'll do this!